frontend/src/pages/404.js

```diff
@@ -2,11 +2,40 @@ import * as React from "react";
 
```

Comment:
for responsive if all the breakpoints are the same, you just need to use the smallest breakpoint  for example `lg:pb-20 md:pb-20 sm:pb-20 mb-80` should be this: `sm:pb-20 mb-80`